src/models/msg.js

```diff
@@ -27,7 +27,7 @@ function Msg(attr) {
 		from: "",
 		id: id++,
 		text: "",
-		time: moment().utc().format("HH:mm:ss"),
```

Comment:
This is removed so that we send the whole timestamp to the client, so people can format it how they like.